docs/developers/contributing.md

```diff
@@ -15,32 +15,32 @@ Active committers and contributors are invited to introduce yourself and request
 
 ## Building and Testing
 
-Firstly, we need to ensure development dependencies are installed. With node and npm installed, after cloning the Chart.js repo to a local directory, and navigating to that directory in the command line, we can run the following:
+Firstly, we need to ensure development dependencies are installed. With node and pnpm installed, after cloning the Chart.js repo to a local directory, and navigating to that directory in the command line, we can run the following:
 
 ```bash
-> npm install
+> pnpm install
 ```
 
 This will install the local development dependencies for Chart.js.
 
 The following commands are now available from the repository root:
 
 ```bash
-> npm run build             // build dist files in ./dist
-> npm run autobuild         // build and watch for source changes
-> npm run dev               // run tests and watch for source and test changes
-> npm run lint              // perform code linting (ESLint, tsc)
-> npm test                  // perform code linting and run unit tests with coverage
+> pnpm run build             // build dist files in ./dist
+> pnpm run autobuild         // build and watch for source changes
+> pnpm run dev               // run tests and watch for source and test changes
+> pnpm run lint              // perform code linting (ESLint, tsc)
+> pnpm test                  // perform code linting and run unit tests with coverage
 ```
 
-`npm run dev` and `npm test` can be appended with a string that is used to match the spec filenames. For example: `npm run dev plugins` will start karma in watch mode for `test/specs/**/*plugin*.js`.
+`pnpm run dev` and `pnpm test` can be appended with a string that is used to match the spec filenames. For example: `pnpm run dev plugins` will start karma in watch mode for `test/specs/**/*plugin*.js`.
 
 ### Documentation
 
 We use [Vuepress](https://vuepress.vuejs.org/) to manage the docs which are contained as Markdown files in the docs directory. You can run the doc server locally using these commands:
 
 ```bash
-> npm run docs:dev
+> pnpm run docs:dev
```

Comment:
Is this still correct then, don't you need to cd into the docs folder and run `pnpm dev` now?

Or does pnpm see the part before the double dot a folder?

Reply:
yes, this works: https://github.com/chartjs/Chart.js/pull/10576#discussion_r940620473. not sure what double dot you're referring to

Reply:
docs**_:_**dev that double dot but your other comment clarified it

Reply:
oh, that's called a colon :smile: I was looking for a `..`

Reply:
Oooh yeah mb, tranated it too litterly 😅